runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified code as specified by [`process`](config.md#process-configuration).
+If `process.args` was not configured, the runtime MUST generate an error.
```

Comment:
That sounds like a subset of the [current `process.args` requirements][1], so it would fall under the [“MAY validate” portion of `create`][2].  Do we need a “MUST validate `process`” (or just `process.args`?) clause in `start`?  My personal preference would be to leave validation up to the caller, who can run [bundle-validation tools][3] whenever they want.  I'm happy to make that an explicit part of the `start` spec, if folks think it would help.  I'll grudgingly make a MUST-validate-whatever clause a part of the `start` spec if that's the maintainer consensus ;).

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/config.md#process
[2]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/runtime.md#create
[3]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/implementations.md#testing--tools